include/javabind/string.hpp

```diff
@@ -162,4 +162,11 @@ namespace javabind
     // helper to extract value
     template <std::string_view const&... items>
     static constexpr auto join_sep_v = join_sep<items...>::value;
+
+    // Remove everything before and including the last colon
```

Comment:
For consistency with the rest of the code, we might want to use comment blocks with `/**` and `*/`, and the Doxygen parameter definition style with `@param`.

I am not sure what *the last colon* is referring to. Perhaps you meant *the last character `ch`*?

Reply:
Updated comment